utils/network.go

```diff
@@ -340,3 +345,41 @@ func writeProcSys(path, value string) error {
 	}
 	return err
 }
+
+// Generate a mac address from the passed in string
+func genLocalMacAddr(src string) net.HardwareAddr {
+	// The 2nd bit set of the first octet makes this MAC address a "locally administered address"
+	// https://en.wikipedia.org/wiki/MAC_address
+	macString := "02"
```

Comment:
The 2 is just one of 6 remaining bits (the first bit is overloaded for uni/multicast). I'm not sure how big the namespace needs to be for these generated mac addresses, but potentially we'll need those extra 6 bits.

Reply:
The rest of the octets leaves a space of 2^40 so I don't think it would be a big deal.  Especially since these don't really need to be unique.  We could just set them all the same and still be fine.  So I don't think the additional complication of using those 6 bits is worth it.

Reply:
okay. but I was looking forward to seeing bitwise operators in golang. maybe next time.